*** Python interpreter version ***

3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 17:14:51) 
[GCC 7.2.0] 

*** Joker Computational Node Allocation ***
Joker 12-13 

*** Dispy Server Startup Messages ***



  "psutil" module is not available;
    node status (CPU, memory, disk and swap space usage) will not be sent to clients

2018-08-21 01:48:33 dispynode - dispynode version: 4.9.1, PID: 23663
2018-08-21 01:48:33 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:33 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:33 pycos - version 4.7.7 with epoll I/O notifier

  "psutil" module is not available;
    node status (CPU, memory, disk and swap space usage) will not be sent to clients

2018-08-21 01:48:33 dispynode - dispynode version: 4.9.1, PID: 2747
2018-08-21 01:48:33 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:33 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:33 pycos - version 4.7.7 with epoll I/O notifier
Starting Initial Optimization
2018-08-21 01:48:40 pycos - version 4.7.7 with epoll I/O notifier
2018-08-21 01:48:40 dispy - dispy client version: 4.9.1
2018-08-21 01:48:40 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:40 dispy - IPv6 may not work without "netifaces" package!
2018-08-21 01:48:40 dispy - Storing fault recovery information in "_dispy_20180821014840"
2018-08-21 01:48:33 dispynode - "joker-c12" serving 48 cpus
4.36 %
Traceback (most recent call last):
  File "/software/development/Build/Anaconda3-4.4.0/envs/python-3.6/bin/dispynode.py", line 197, in _dispy_job_func
    __dispy_job_name, globals())
  File "<string>", line 1, in <module>
  File "<string>", line 5, in compute
  File "./botProto1.py", line 547, in backtest
    self.performance = self.get_performance(self.trade_info,[start_seconds,end_seconds,patt_cnt,corr_pats])
  File "./botProto1.py", line 686, in get_performance
    apr = 100 * (np.exp((float(252) * np.log(equity[-1] / equity[0])) / t_elapsed) - 1)
IndexError: index -1 is out of bounds for axis 0 with size 0

Traceback (most recent call last):
  File "/var/spool/slurmd/job379610/slurm_script", line 1223, in <module>
    peak_param, pattern_error, pip_res, perf=bot.backtest(data,parameters,web_up=False)
  File "/var/spool/slurmd/job379610/slurm_script", line 492, in backtest
    performance = opt.search()
  File "/var/spool/slurmd/job379610/slurm_script", line 1129, in search
    {'peak': retval[0], 'error': retval[1],
TypeError: 'NoneType' object is not subscriptable
